### DISCUSSION TITLE
Can't Verify Because of Profile

### DISCUSSION THREAD

**Discussion by DBTeacherVictorLu** [community/community#52642] [GitHub Education]
I have tried to update my profile several times but the verification would not let me.  My bio only has 5 characters left and I filled in whatever else I can fill in. As for my name, that is it.  Maybe my last name is too short?

This is the response I got from the rejection:

> You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

**Comment by fily-gif** [comment]
same here, but in my case I do not want to paste my full name into my github profile for privacy reasons.

**Comment by MrZebarth** [comment]
I'm not getting verified because it's claiming my staff ID card doesn't list my educational institution on it, despite it being very clearly written. I've never had any issues getting verified in the past, but now I've been rejected 3 times in a row. I'm assuming they are using an automated system, but it hasn't "learned" how to read my staff ID yet. Any help would be greatly appreciated!

**Comment by samus-aran** [comment]
Hi @DBTeacherVictorLu 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Hope this helps! Let us know how you get along 👍
